docs/content/integrations/embedded-elt/dlt.mdx

```diff
@@ -100,11 +141,18 @@ $ tree -a
 └── requirements.txt   # can be removed
 ```
 
-## 3. Define environment variables required by `dlt`
+---
 
-This integration manages connections and secrets using environment variables as `dlt`. The `dlt` library is able to "magically" infer required environment variables used by its sources and resources. See [Secrets and Configs](https://dlthub.com/docs/general-usage/credentials/configuration) for more information.
+## Step 3: Define dlt environment variables
 
-In this example, the `github_reactions` source requires a GitHub access token, and the _snowflake_ destination requires database connection details. This results in the following required environment variables:
+This integration manages connections and secrets using environment variables as `dlt`. The `dlt` library can infer required environment variables used by its sources and resources. Refer to [dlt's Secrets and Configs](https://dlthub.com/docs/general-usage/credentials/configuration) documentation for more information.
+
+In the example we've been using:
+
+- The `github_reactions` source requires a GitHub access token
+- The Snowflake destination requires database connection details
+
+This results in the following required environment variables:
```

Comment:
Where should users be defining these? In an `.env`? Elsewhere? We should be specific here if we can.